### DISCUSSION TITLE
Basic Tone and Local Adjustments on same knob of X Touch mini

### DISCUSSION THREAD

**Discussion by zerocoolisking** [rsjaffe/MIDI2LR#655] [Q&A] [answered]
Is it possible to set Basic Tone Exposure and Local Adjustments Exposure on same knob, depending if some mask is open or just a image?

**Comment by rsjaffe** [comment, marked as answer]
Yes, use profiles. See https://github.com/rsjaffe/MIDI2LR/wiki/Profiles. Assign one profile to **Masking**that uses local adjustments for the key and one profile to **Develop** that uses basic tone.
